app/helpers/field_validation_helper.rb

```diff
@@ -1,6 +1,19 @@
 # frozen_string_literal: true
 
 module FieldValidationHelper
+  def validate_field_response_length(page, fields)
+    invalid_fields = []
+    fields.each do |field|
+      next unless params[field]
```

Comment:
so... params are not in the session and are always string?
Field is from that word array in each controller... so these match?

Reply:
The params hash has `HashWithIndifferentAccess`, so we can use either strings or symbols to access